src/ec/packages/ec-component-message/message.story.js

```diff
@@ -15,77 +17,52 @@ import dataWarning from './demo/data--warning';
 import message from './ecl-message.html.twig';
 import notes from './README.md';
 
-const formatIcon = data => {
-  data.icon.path = defaultSprite;
-  data.close.icon.path = defaultSprite;
+const prepareMessage = data => {
+  const variantList = {};
+  variantList[data.variant] = data.variant;
+  data.variant = select(
+    'variant',
+    variantList,
```

Comment:
i don't neessarily understand this, @papegaill ..wouldn't be enough to create the select like this?
`data.variant = select('variant', [data.variant], data.variant); `